l2-contracts/contracts/ConsensusRegistry.sol

```diff
@@ -0,0 +1,278 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.20;
+
+import {EfficientCall} from "@matterlabs/zksync-contracts/l2/system-contracts/libraries/EfficientCall.sol";
+
+/// @title ConsensusRegistry
+/// @dev A contract to manage consensus nodes and committees.
```

Comment:
Please also expend the description. What are committees here? What is their role? How this contract should be used?
```suggestion
/// @author Matter Labs
/// @custom:security-contact security@matterlabs.dev
/// @title ConsensusRegistry
/// @dev A contract to manage consensus nodes and committees.
```

Reply:
Docs added: https://github.com/matter-labs/era-contracts/pull/555/commits/5d80cbc9c41fef107d408cca9bf5761f16be63a4